official/docs/ruby/orders/create_order.rb

```diff
@@ -0,0 +1,21 @@
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_API_KEY']
+
+order = EasyPost::Order.create(
+  {
+    to_address: {
+      id: 'adr_...',
+    },
+    from_address: {
+      id: 'adr_...',
+    },
+    shipments: [
+      { parcel: { predefined_package: 'FedExBox', weight: 10.2 } },
+
+      { parcel: { predefined_package: 'FedExBox', weight: 17.5 } },
```

Comment:
Let's split up JSON to one item per line

```suggestion
      { 
        parcel: { 
          predefined_package: 'FedExBox', 
          weight: 10.2
        }
      },

      { parcel: { predefined_package: 'FedExBox', weight: 17.5 } },
```